APLSource/A2K/∆VGET.aplf

```diff
@@ -1,10 +1,10 @@
  ∆VGET←{
      ⍺←1
      Exec←(2147483647×~⍺)∘(⍬⍴⎕RSI).(86⌶)
-     (varname default)←w←⊆⍵
+     (varname default)←w←{0=10|⎕DR ⍵:⍵ 0 ⋄ 2≠≢⍵: 2↑(⊆⍵),⊂0 ⋄ ⍵} ⍵
```

Comment:
* `2↑` does not correctly model LENGTH ERROR in APL+Win's `⎕VGET 'foo' 'bar' 'baz'`.
* Returns `0` if missing var and no default given

How about `w←⊂⍣(326≠⎕DR ⍵)⊢⍵`